apps/ui/src/i18n/locale/en/enTranslations.json

```diff
@@ -101,6 +101,10 @@
 	},
 	"errors": {
 		"http": {
+			"403": {
+				"title": "Unauthorized.",
+				"message": "Sorry, looks you're not authorized to access the resource you're trying to access."
```

Comment:
I think `you're trying to access` is maybe redundant, could just be `... not authorized to access this resource`